build/platform.device.mak

```diff
@@ -12,4 +12,9 @@ EXE = elf
 
 .PHONY: %_flash
 %_flash: %.bin
-	dfu-util -i 0 -a 0 -s 0x08000000:leave -D $<
+	@echo "DFU     $@"
+	@echo "INFO    About to flash your device. Please plug your device to your computer"
+	@echo "        using and USB cable and press the RESET button the back of your device."
```

Comment:
Typo @Ecco : "using and USB" => "using a USB"

Reply:
Indeed… :-)